proposals/UX.md

```diff
@@ -0,0 +1,45 @@
+# Application form for starting a SIG
+
+## What will be the name of the SIG
+
+User Experience SIG:  **UX-SIG**
+![Creativity](https://positivepsychology.com/wp-content/uploads/boost-your-creativity.jpg)
+
+## Which two persons will act as SIG Lead?
+
+- Jesus Garcia Gonzalez
+- Sven van der Burg
+
+
+## What is the SIG's mission?
+
+<!-- The mission of your SIG should contribute to the organization's mission. -->
+
+The SIG aims to start slowly expanding the knowledge and the importance of Human-centric design and best User Experience techniques within our organization and research projects, especially when developing software. 
+Visualization and presentation are also substantial parts of the SIG, improving seamless guidelines and keeping an eye on consistency and best practices following the eScience branding and communication guidelines. During the SIG meetings, the focus will be on providing design critiques to existing projects, analyzing the design and procedures, and giving feedback to the team members.
+
+## What GitHub repository will the SIG use?
+To be created
+
+## What Office group will the SIG use?
+UX-SIG, ux@esciencecenter.nl (to be created)
+
+## Plans for period until the end of April 2020
+
+- **Design Critiques for current projects**:  encouraging a positive culture to improve projects. Analyze the design of the project's user journey, usability, accessibility, interaction, visualization, and presentation, and give feedback to team members. While making sure of keeping  a **Clear scope for the conversation",   "Agreed-upon design objectives for the work."  and motivating "Conversation rather than command." One project per Meeting. 
+
+- **Proportionate UX tools and resources**: from tools for ideation, wireframing and prototyping, to resources for best practices and design patterns.
+ 
+## What are the expected outputs of the proposed SIG?
+
+- **Design Thinking workshops**:  a Design Thinking workshop is a hands-on, activity-based session built around the Design Thinking process.
+
+- **Contributions to eScience Center Guide** on the following topics:
+	-   Research / User testing
+	-   Persona / Empathy map
+	-   Information architecture
+	-   Wireframe creation
+	-   Style guide creation (or UI template)
+	-   Visual design
+	-   Prototyping
+ 
```

Comment:
```suggestion
## Plans for period until the end of April 2020

- **Bring Your Own Project Design Critique sessions**:  During a SIG meeting we analyze the design of one NLeSC project. We analyze the design of the project's user journey, usability, accessibility, interaction, visualization, and presentation, and give feedback to team members. In this session we will make sure of keeping  a "Clear scope for the conversation",   "Agreed-upon design objectives for the work."  and motivating "Conversation rather than command." 

- **Learn about UX tools and resources**: From tools for ideation, wireframing and prototyping, to resources for best practices and design patterns.
 
## What are the expected outputs of the proposed SIG?

- **Design Thinking workshops**:  a Design Thinking workshop is a hands-on, activity-based session built around the Design Thinking process.

- **Contributions to eScience Center Guide** on the following topics:
	-   Research / User testing
	-   Persona / Empathy map
	-   Information architecture
	-   Wireframe creation
	-   Style guide creation (or UI template)
	-   Visual design
	-   Prototyping
 
```